src/finder/RawFileFinder.ts

```diff
@@ -2,19 +2,35 @@ import { Finder, UmlContent } from './Finder';
 
 export class RawFileFinder implements Finder {
   private readonly URL_REGEX = /^.*\.(plantuml|pu|puml)(\?.*)?$/;
+  private readonly INCLUDE_REGEX = /!include\s+(.*\.(plantuml|pu|puml))/g;
 
   canFind(webPageUrl: string): boolean {
     return this.URL_REGEX.test(webPageUrl);
   }
 
   async find(webPageUrl: string, $root: JQuery<Node>): Promise<UmlContent[]> {
     const $texts = $root.find('pre');
+    const dirUrl = webPageUrl.replace(/\/[^/]*\.(plantuml|pu|puml)(\?.*)?$/, '');
     const result = [];
     for (let i = 0; i < Math.max($texts.length, 1); i++) {
       const $text = $texts.eq(i);
-      const content = $text.text();
-      if (content.indexOf('@startuml') > -1) result.push({ $text, text: content });
+      let content = $text.text();
+      if (content.indexOf('@startuml') < 0) continue;
+      while (true) {
```

Comment:
We can use `for (;;)` instead of `while (true)` or `while (match = this.INCLUDE_REGEX.exec(content))`